source/grid/grid_in.cc

```diff
@@ -1404,10 +1404,11 @@ GridIn<dim, spacedim>::read_msh(std::istream &in)
   Assert(tria != nullptr, ExcNoTriangulationSelected());
   AssertThrow(in, ExcIO());
 
-  unsigned int n_vertices;
-  unsigned int n_cells;
-  unsigned int dummy;
-  std::string  line;
+  unsigned int                      n_vertices;
+  unsigned int                      n_cells;
+  unsigned int                      dummy;
+  std::string                       line;
+  std::array<std::map<int, int>, 4> tag_maps;
```

Comment:
Can you document what this maps from and to, and what the `4` stands for? I suspect it's the dimension of the object or some such?